app/client/lib/ethereum/observeCustomContracts.js

```diff
@@ -76,29 +76,39 @@ observeCustomContracts = function(){
         @method added
         */
         added: function(newDocument) {
+            // Disables it by default
+            CustomContracts.update(newDocument._id, {$set: {
+                disabled: true
+            }});
 
+            // check for logs
+            addLogWatching(newDocument);
```

Comment:
You can actually just leave the code as it was, and add the second balance check before ultimately disabling it.

Adding the watch logs is surely not necessary if there is not code at the address.